Source/Scene/Material.js

```diff
@@ -298,6 +301,28 @@ import when from '../ThirdParty/when.js';
          */
         this.translucent = undefined;
 
+        /**
+         * The {@link TextureMinificationFilter} to apply to this material.
+         * The default value is {@link TextureMinificationFilter.LINEAR}.
+         *
+         * To take effect, this property must be set before the texture is created.
+         *
+         * @type {TextureMinificationFilter}
+         * @default {@link TextureMinificationFilter.LINEAR}
+         */
+        this.minificationFilter = defaultValue(options.minificationFilter, TextureMinificationFilter.LINEAR);
```

Comment:
These options should be documented in the `Material` constructor.

If these options only take effect before textures are created maybe they should only be documented in the constructor and not here.